esp32/PermanentMarker36.c

```diff
@@ -0,0 +1 @@
+../../components/ugfx/PermanentMarker36.c
```

Comment:
hmm it'd be nice to avoid this but it's consistent with how we handle other fonts ;)